integration-cli/docker_api_stats_test.go

```diff
@@ -231,6 +231,9 @@ func (s *DockerSuite) TestApiStatsContainerNotFound(c *check.C) {
 func (s *DockerSuite) TestApiStatsContainerGetMemoryLimit(c *check.C) {
 	testRequires(c, DaemonIsLinux)
```

Comment:
Obviously you are right: .\docker_api_stats_test.go:232: undefined: memoryLimitSupport